QuickLayout.Touch/DebugTrace.cs

```diff
@@ -1,31 +1,30 @@
-using System;
-using System.Diagnostics;
-using MvvmCross.Platform.Platform;
+//using System;
+//using System.Diagnostics;
 
-namespace QuickLayout.Touch
-{
-    public class DebugTrace : IMvxTrace
-    {
-        public void Trace(MvxTraceLevel level, string tag, Func<string> message)
-        {
-            Debug.WriteLine(tag + ":" + level + ":" + message());
-        }
+//namespace QuickLayout.Touch
```

Comment:
And this?

Reply:
Yes, I forgot to delete that file. Would you like me to do so?

Reply:
Yes, totally. I've forgot to delete the file. Would you like me to do so?

Reply:
MvvmCross will automatically marshal RaisePropertyChanged to the UI thread, unless you tell it not to.

Reply:
Yeah we may as well just remove anything commented out/not needed. Can you also rebase on the latest in master? I just went ahead and fixed the build via #58 so want to get this on top of that. Thanks!

Reply:
Yes, sure. I'll do that as soon as I can.